cmd/readme_test/readme.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
okay. since it is not impacting the code in anyway, and it serves a specific purpose, am fine with that.